citesphere/src/main/java/edu/asu/diging/citesphere/core/model/bib/impl/CollectionResult.java

```diff
@@ -0,0 +1,147 @@
+package edu.asu.diging.citesphere.core.model.bib.impl;
+
+import java.util.List;
+import edu.asu.diging.citesphere.core.model.bib.ICitation;
+import edu.asu.diging.citesphere.core.model.bib.ICitationCollection;
+import edu.asu.diging.citesphere.core.model.bib.ICitationGroup;
+import edu.asu.diging.citesphere.core.model.bib.ICollectionResult;
+
+public class CollectionResult implements ICollectionResult {
```

Comment:
since this isn't a core model class (that is persistent into the db), this should be in the model package